views/index.html

```diff
@@ -306,6 +306,20 @@
 {{/unless}}
 
 <script src="{{static}}/js/vendor/jquery-1.11.0.min.js"></script>
+<script>
+  // Load libraries from a server config file
+  // This method is used so we can both expand vars, and support <script> in the json strings
+  window.libraries = window.libraries || [];
+  $.ajax({
```

Comment:
Why load this over ajax if it's baked into the root? Either it should be loaded into the markup directly - as in part of the view via a template var or it should be lazy loaded after jsbin is ready.